docs/source/examples/capacity_expansion_tutorial.ipynb

```diff
@@ -0,0 +1,909 @@
+    "    ax.fill_between(hours, \n",
+    "                    y1=0, \n",
+    "                    y2=model.results['NaturalGas_Conv'].values, \n",
+    "                    color='tab:orange', \n",
+    "                    label='Natural Gas')\n",
+    "    ax.fill_between(hours, \n",
+    "                    y1=model.results['NaturalGas_Conv'].values, \n",
+    "                    y2=model.results['Battery'].values+model.results['NaturalGas_Conv'].values, \n",
+    "                    color='tab:green', \n",
+    "                    label='Battery discharge')\n",
+    "    ax.fill_between(hours, \n",
+    "                    y1=0, \n",
+    "                    y2=model.results['Battery_charge'].values, \n",
+    "                    color='tab:pink', \n",
+    "                    label='Battery charge')\n",
+    "    ax.plot(hours, model.net_demand, color='cyan', label='Net Demand')\n",
+    "    ax.set_xlim(0,48)\n",
+    "    # ax.set_ylim(0,5.5)\n",
+    "    ax.legend(loc='upper left')\n",
+    "    ax.set_ylabel(\"Demand [MW]\")\n",
+    "    ax.set_xlabel(\"Time [hr]\")\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding renewable energy to a model\n",
+    "\n",
+    "\n",
+    "The previous model used natural gas and batteries to meet demand. What if we wanted to include wind as well?\n",
+    "\n",
+    "#### How the Capacity Expansion model tests points\n",
+    "\n",
+    "By default, the capacity expansion model will choose a set of random values, $\\mathcal{X}_i$, between 0 and 1 (inclusive), for each\n",
+    "technology in the technology list where $\\mathcal{X}$ represents a fraction of the maximum energy demand (up to 100%). The capacity of \n",
+    "each technology is calculated by \n",
+    "\n",
+    "$\\mathcal{C}_i = \\mathcal{X}_i\\cdot \\max\\left(\\mathcal{E}\\right)$\n",
+    "\n",
+    "where $\\mathcal{E}$ is the energy demand time series.\n",
+    "\n",
+    "Therefore, the maximum possible capacity for any technology is equal to the maximum demand energy demand. This is fine for dispatchable technologies because we can always choose to turn a dispatchable technology on or off. This is not true for variable renewable energy sources,\n",
+    "such as wind. If we only built enough wind turbine capacity to match the peak demand value we\n",
+    "\n",
+    "1. cannot be sure that the peak production will align with peak demand and\n",
+    "2. it is extremely unlikely that peak production will be sustained for any amount of time.\n",
+    "\n",
+    "To get around this, we can specify a peak value greater than one. This value can be as high as you want -- but best results are usually found when the maximum capacity for a renewable source is \n",
+    "\n",
+    "$\\max\\left(\\mathcal{X}\\right) = \\frac{1}{CF}$,\n",
+    "\n",
+    "where $CF$ is the average capacity factor."
+   ]
```

Comment:
Would "net capacity" be a better expression than "capacity" here? "Capacity" may be confused with the original definition; maximum power output of an individual power plant.

Reply:
I'm not sure what you mean... this is *is* the maximum power output of an individual plant. What I'm trying to describe is the range of values that $\mathcal{X}$ can take on -- which is a fraction of the maximum of the energy demand. $\max{\mathcal{E}}$ is equivalent to the maximum instantaneous power demand.

Reply:
Okay then ignore my initial comment.

This may be a nitpick but you could edit the first line to "By default, the capacity expansion model samples a set of values, X_i, ..." to clarify the model is picking values based on some algorithm criteria rather than purely randomly.